Dockerfile

```diff
@@ -195,6 +195,12 @@ RUN fname="go-ipfs_${IPFS_VERSION}_${TARGETOS:-linux}-${TARGETARCH:-amd64}.tar.g
   tar -C ${tmpout} -xzf "${fname}" go-ipfs/ipfs && \
   mv ${tmpout}/go-ipfs/ipfs /out/bin/ && \
   echo "- IPFS: ${IPFS_VERSION}" >> /out/share/doc/nerdctl-full/README.md
+ARG TINI_VERSION
+RUN tiniFilename=tini-static-${TARGETARCH:-amd64} && \
+        tiniSumfile="${tiniFilename}.sha256sum" && \
+        curl -o "${tiniFilename}" -fSL https://github.com/krallin/tini/releases/download/${TINI_VERSION:-v0.19.0}/tini-static-${TARGETARCH:-amd64} && \
+        curl -o "${tiniSumfile}" -fSL https://github.com/krallin/tini/releases/download/${TINI_VERSION:-v0.19.0}/tini-static-${TARGETARCH:-amd64}.sha256sum && \
+        grep "${tiniFilename}" "${tiniSumfile}" | sha256sum -c && cp ${tiniFilename} /out/bin/tini
```

Comment:
Please add version and license to  /out/share/doc/nerdctl-full/README.md (see other lines)